torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -600,6 +600,7 @@ def run(args: List[str], **kwargs: Any) -> Tuple[CompletedProcessType, str]:
                     args,
                     stdout=f_stdout_stderr,
                     stderr=subprocess.STDOUT,
+                    check=True,
```

Comment:
I think this specific forward call caused issues that causes tests to file. Should probably have a noqa added to it.